hardware/DomoticzHardware.cpp

```diff
@@ -26,15 +26,32 @@ bool CDomoticzHardwareBase::CustomCommand(const uint64_t /*idx*/, const std::str
 	return false;
 }
 
-bool CDomoticzHardwareBase::Start()
+void CDomoticzHardwareBase::Start()
```

Comment:
please keep the boolean as also with the stop.
If it is not used, ok, but at least we also return true/false.